storage/build.gradle

```diff
@@ -6,7 +6,7 @@ dependencies {
   implementation project(':util')
 
   compile 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
-  compile group: 'org.mapdb', name: 'mapdb', version: '3.0.7'
+  compile 'org.mapdb:mapdb'
```

Comment:
Any reason these two use compile not implementation?

Reply:
As of yet unknown. I've got another task to address the `compile` and `runtime` usages, as they are deprecated in favour of `api`, `implementation`, `compileOnly` and `runtimeOnly` depending on precisely how they are needed (plus as those are used elsewhere, it'll keep these `build.gradle` consistent).